python-package/lightgbm/plotting.py

```diff
@@ -441,8 +481,9 @@ def create_tree_digraph(booster, tree_index=0, show_info=None, precision=None,
     show_info : list of strings or None, optional (default=None)
         What information should be shown in nodes.
         Possible values of list items:
-        'split_gain', 'internal_value', 'internal_count', 'internal_weight', 'leaf_count', 'leaf_weight'.
-    precision : int or None, optional (default=None)
+        'split_gain', 'internal_value', 'internal_count', 'internal_weight',
+        'leaf_count', 'leaf_weight', 'data_percentage'.
+    precision : int or None, optional (default=3)
```

Comment:
Could you expand this documentation please? What are the units and what is the implication of setting `None`?

Reply:
@jameslamb This is nice catch, but I think this is beyond of this PR. `precision` was introduced in #1424 and is used in several functions, so I'll address your comment in a separate PR after merging this one.

Reply:
makes sense!